lib/scaffold/create.js

```diff
@@ -19,8 +19,8 @@ var BASE_PACKAGE = {
   license: 'MIT',
   readme: 'README.md',
   dependencies: {
-    'bitcore-lib-zen': 'HorizenOfficial/bitcore-lib-zen',
-    'bitcore-node-zen': 'HorizenOfficial/bitcore-node-zen'
+    'bitcore-lib-zen': 'HorizenOfficial/bitcore-lib-zen#zendoo-support',
```

Comment:
Yes, in fact we will start working with tags, so instead of #zendoo-support we will add something like #v1.0.0 or whatever tag we have to use (we will be dumping mayor versions in this release)